homeassistant/components/media_source/models.py

```diff
@@ -58,19 +57,20 @@ async def async_browse(self) -> BrowseMediaSource:
             base = BrowseMediaSource(
                 domain=None,
                 identifier=None,
-                media_class=MEDIA_CLASS_DIRECTORY,
-                media_content_type=MEDIA_TYPE_CHANNELS,
+                media_class=MEDIA_CLASS_APP,
+                media_content_type=MEDIA_TYPE_APPS,
                 title="Media Sources",
                 can_play=False,
                 can_expand=True,
-                children_media_class=MEDIA_CLASS_CHANNEL,
+                children_media_class=MEDIA_CLASS_APP,
             )
             base.children = [
                 BrowseMediaSource(
                     domain=source.domain,
                     identifier=None,
-                    media_class=MEDIA_CLASS_CHANNEL,
-                    media_content_type=MEDIA_TYPE_CHANNEL,
+                    media_class=MEDIA_CLASS_APP,
+                    media_content_type=MEDIA_TYPE_APP,
+                    thumbnail=f"https://brands.home-assistant.io/{source.domain}/logo.png",
```

Comment:
This will serve a fallback "logo" placeholder.
Should we use the 404 throwing version instead? (Which we use on the integrations page I believe).

```suggestion
                    thumbnail=f"https://brands.home-assistant.io/_/{source.domain}/logo.png",
```